moja-kuhinjica-frontend/src/pages/editProfile/[id].tsx

```diff
@@ -79,43 +117,47 @@ const EditProfilePage = (): JSX.Element => {
                             />
                             <FormInput
                                 register={register}
                                 errors={errors}
-                                name="email"
-                                src={email}
-                                placeholder="Email"
+                                name="surname"
+                                src={profile}
+                                placeholder="Prezime"
                                 type="text"
                                 validationSchema={{
-                                    required: 'email is required',
+                                    required: 'surname is required',
                                     pattern: {
-                                        value: /^\w+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w{2,3})+$/,
-                                        message: 'invalid email value',
+                                        value: /[A-Za-z]/,
+                                        message: 'invalid surname value',
                                     },
                                 }}
+                                defaultValue={user?.surname}
                             />
                             <FormInput
                                 register={register}
                                 errors={errors}
-                                name="password"
-                                src={password}
-                                placeholder="Šifra"
-                                type="password"
+                                name="email"
+                                src={email}
+                                placeholder="Email"
+                                type="text"
                                 validationSchema={{
-                                    required: 'password is required',
+                                    required: 'email is required',
                                     pattern: {
-                                        value: /^(?=.*[A-Za-z])(?=.*\d)[A-Za-z\d]{8,}$/,
-                                        message: 'invalid password value',
+                                        value: /^\w+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w{2,3})+$/,
+                                        message: 'invalid email value',
                                     },
                                 }}
+                                defaultValue={user?.email}
                             />
+
                             <FormInput
                                 register={register}
                                 errors={errors}
-                                name="telephoneNumber"
+                                name="phoneNumber"
                                 src={mobile}
-                                placeholder="Broj telefona"
+                                placeholder=""
                                 type="text"
```

Comment:
This input also needs to be changed, cause now you can enter letters in phone number field